components/list/demo/list-item-sample.js

```diff
@@ -0,0 +1,50 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { ListItemCheckboxMixin } from '../list-item-checkbox-mixin.js';
+//import { ListItemDragMixin } from '../list-item-drag-mixin.js';
+
+class ListItemSample extends ListItemCheckboxMixin(LitElement) {
+	static get properties() {
```

Comment:
I think `disabled`, `key`, and `role` should be coming from the mixin(s).  Presumably the breakpoint stuff is specific to our built-in list-item layout and isn't going into the mixin, but I am not sure.  (@m6jones , what did you intend here?)

Reply:
`breakpoint` I forgot about that stuff!

Reply:
`disabled` `key` should be part of the `checkboxMixin`

I would leave the `role` and `breakpoint` logic here for now.

Reply:
I feel pretty strongly that this should not be specified by consumers.  Will look forward to iterating on this.

Reply:
Yeah, this will come later closer to the end of of all these changes.

Reply:
I promise that a consumer will never have to touch roles. You know me I believe strongly in that too :D

Reply:
I've added a TODO note and removed role and breakpoint altogether for now to defer for later :)